content/test/integration/TestServer.ts

```diff
@@ -61,8 +61,8 @@ export class TestServer extends Server {
         return this.client.fetchHistory()
     }
 
-    getDeployments(filters?: DeploymentFilters): Promise<ControllerDeployment[]> {
-        return this.client.fetchAllDeployments(filters, DeploymentFields.POINTERS_CONTENT_METADATA_AND_AUDIT_INFO)
+    getDeployments(filters?: DeploymentFilters, sort?: DeploymentSorting): Promise<ControllerDeployment[]> {
```

Comment:
Maybe this is a good opportunity to make parameters an object, like we did before?